EM - repeat hackytaq

ericmjl opened the pull request.

ericmjl committed:
started creating hackytaq repeat report

ericmjl committed:
Wrote in goals

ericmjl commented:
@Starnite you can start reviewing as we go along.

ericmjl committed:
added details of running gel

turns out there may be issues w/sequencing because of mixed product;
this is just a check to make sure.

ericmjl committed:
added gel interpretation and gel image

ericmjl committed:
added addendum to prior run

for future reference.

ericmjl committed:
Updated with follow-on experiment

Starnite commented on experiments/20161007-hackytaq-repeat/20161007-hackytaq-repeat.md:
```diff
@@ -0,0 +1,57 @@
+
+## Interpretation
+
+One look at the gel and I realized what was going wrong. The melting temperature was incorrect. I had used 68ºC in this PCR round; in fact, it should have been 60ºC.
```
For all previous experiments we've been using Tm of 68, including the hacky-taq that I ran and the PCR we used to generate the low-mutation PB2 part 1 fragments for the cpec assembly.

ericmjl replied:
That's weird, hmmm...

Referencing back on [this](https://github.com/ericmjl/protein-systematic-characterization/blob/master/experiments/20160805-mutagenesis/20160829-gradient-pcr.md) experiment, 68ºC yielded no PCR product for part 2, which is the hacky-taq PCR product for primer set EM-27/28, whereas at 60ºC the correct product shows up.  According to that gradient PCR, EM-30/29 should be run at 68ºC.

Are you sure that the `mytaq red` PCR program was run using 68ºC, and not 60ºC? If you're not sure about it, may be good to start a new fork off master and update [that note](https://github.com/ericmjl/protein-systematic-characterization/blob/master/experiments/20160920-Taq-Mn-mutagenesis/20160912-mgcl2-taq-series.md) with a little bullet point stating that the Tm wasn't recorded down. (No worries that it wasn't recorded down that one time, these things happen, but it'll be important to note this down so that we remember for future reference.)

ericmjl replied:
@Starnite: I think I have some memory of what happened - that particular run, it was very likely 58ºC, not 68ºC. We chatted about it and decided to low-ball the annealing temperature, just to be conservative.

Nonetheless, the PCR conditions were not noted down, so I think it's best to add a note to the report indicating so.

ericmjl committed:
updated with gel of PCR re-run

After re-running the gel, obtained correct PCR bands.

ericmjl committed:
added sequencing order + DNA concentrations

ericmjl committed:
Merge branch 'master' into em-repeat-hackytaq

ericmjl commented:
Current status: just sent in PCR products for sequencing. We will get back the sequencing results tomorrow morning. I hope it'll be correct this time round.

ericmjl committed:
added PCR program + sequencing details

ericmjl committed:
added sequencing results

ericmjl committed:
Added script for creating alignment.

ericmjl committed:
ran alignment of sequences with victoria-PB2

ericmjl committed:
Finalized report discussion

ericmjl committed:
re-ran hackytaq notebook

ericmjl committed:
Merge branch 'master' into em-repeat-hackytaq

ericmjl committed:
added repeat sequencing data

ericmjl committed:
Started report on final try of hackytaq PCR sequencing

ericmjl committed:
updated report with additional info

1. gel purification
2. sequencing

ericmjl committed:
added sequencing data and comments on sequencing results

ericmjl committed:
updated hackytaq notebook with latest sequencing data

ericmjl committed:
updated lab report on how I manipulated the files for analysis.

ericmjl commented:
ping @Starnite: please review.

Summary of this set of experiments: 
1. **definitely want to gel purify before sequencing**. All of the samples had good CRL scores (see Genewiz screenshot), much better than my previous two attempts, and moderately better than your first experimental run.
2. I can see why 0.5 mM Mn<sup>2+</sup> is used, though I think this deserves one more replication to be sure.

Starnite reviewed (approved):
Shall we clone PCR products from this run for another pol assay? We need more randomly mutated PB2 part 2 anyway.

ericmjl commented:
@Starnite: This time round, I sent in all 10 µL of the PCR product for sequencing, so there's none left ☹️. That's because the DNA concentrations were low enough necessitating all of the eluted product being sent in.

If you look at the temp-hackytaq notebook, you'll notice that the replication run had much fewer mutations than before. Given that, I'm not quite yet confident in the number of mutations that will show up, so I think we'll need a replication run of the hackytaq PCR reactions to get a more reliable estimate of which concentration of Mn<sup>2+</sup> to use.

ericmjl committed:
Merge branch 'master' into em-repeat-hackytaq

ericmjl committed:
started note for hackytaq repeat

ericmjl committed:
finished gel extraction & sent for sequencing

ericmjl committed:
updated 20161018 run of hackytaq with data, notebook, and analysis

@Starnite: Once again, I think gel purification is going to be
necessary. If I get to start my own research group, I’m so totally
getting a gel purification robot!

ericmjl commented:
ping @Starnite: let's **not** merge this in yet. I think it's a nice mini-experiment that can go on the side whenever there's downtime, and I can see the data coming in handy with the Bayesian analysis paper.

ericmjl committed:
Merge branch 'master' into em-repeat-hackytaq

ericmjl committed:
started hackytaq repeat in duplicate

ericmjl committed:
updated with gel + gel extraction

ericmjl committed:
added DNA concentration measurements & sequencing order

ericmjl committed:
updated with sequencing results data

ericmjl committed:
updated lab report

ericmjl committed:
re-ran notebook with updated data

ericmjl commented:
@Starnite I think I'd like to merge in. 

After chatting with Stuart (BioMicroCenter), I think we can take this off the priority list. It was a fun experiment to have run, nonetheless.
